terraform/pubsub.tf

```diff
@@ -245,6 +245,28 @@ resource "google_pubsub_subscription" "default" {
   }
 }
 
+resource "google_pubsub_subscription" "json" {
+  project = var.project_id
+
+  name  = "${var.prefix_name}-bq-json-sub"
+  topic = google_pubsub_topic.default.name
+
+  bigquery_config {
+    table            = format("${google_bigquery_table.raw_events_table.project}:${google_bigquery_table.raw_events_table.dataset_id}.${google_bigquery_table.raw_events_table.id}")
```

Comment:
I think this should be "${google_bigquery_table.raw_events_table.project}.${google_bigquery_table.raw_events_table.dataset_id}.${google_bigquery_table.raw_events_tabletable_id}", see example here https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/pubsub_subscription.html#example-usage---pubsub-subscription-push-bq. @pdewilde

Reply:
Thank you, that was a really good catch!